packages/gestalt/src/Table.js

```diff
@@ -16,18 +16,38 @@ import TableSortableHeaderCell from './TableSortableHeaderCell.js';
 import { TableContextProvider } from './contexts/TableContext.js';
 
 type Props = {|
+  /**
+   * Must be instances of Table.Header, Table.Body, and/or Table.Footer components. See the [Subcomponent section](https://gestalt.pinterest.systems/table#Subcomponents) to learn more.
+   */
   children: Node,
+  /**
+   * Label for screen readers to announce Table.
+   */
   accessibilityLabel: string,
+  /**
+   * Specify a border width for table: "sm" is 1px.
+   */
   borderStyle?: 'sm' | 'none',
+  /**
+   * Use numbers for pixels: maxHeight={100} and strings for percentages: maxHeight="100%".
+   */
   maxHeight?: number | string,
+  /**
+   * Specify how many columns from the start of the Table should be sticky when scrolling horizontally. See the [sticky column](https://gestalt.pinterest.systems/table#Sticky-Column), the [multiple sticky columns](https://gestalt.pinterest.systems/table#Multiple-sticky-columns), the [sticky header and columns](https://gestalt.pinterest.systems/table#Sticky-header-and-sticky-columns), the [expandable row with sticky columns](https://gestalt.pinterest.systems/table#Table-Row-Expandable-with-Sticky-Columns), and the [sortable header cells with sticky columns](https://gestalt.pinterest.systems/table#Sortable-header-cells-with-sticky-columns) variants for details.
```

Comment:
You can leave off the article after the first list item: "the foo, bar, baz, and foobar variants…"

Reply:
DONE